js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -503,7 +611,7 @@ const KeyboardNavigationController = core.ViewController.inherit({
 
         if(this._focusCell($cell, !nextCellInfo.isHighlighted)) {
             if(!this._isRowEditMode() && isEditingAllowed) {
-                this._editingController.editCell(this.getVisibleRowIndex(), this._focusedCellPosition.columnIndex);
+                setTimeout(() => this._editFocusedCell());
```

Comment:
Why is setTimeout here?

Reply:
IOS has problems with focus inside setTimeout.

Reply:
I agree, the timeout is redundant here.